python/src/worker/tasks/cluster_cells.py

```diff
@@ -41,7 +44,33 @@ def _format_result(self, raw):
                     "cellIds": list(view.map(int)),
                 }
             )
-        return [Result(json.dumps(cell_set), cacheable=False)]
+        
+        return cell_set
+    
+    def _update_through_api(self, cell_set_object):
+        r = requests.patch(
+            f"{config.API_URL}/v1/experiments/{self.request['experimentId']}/cellSets",
+            headers={
+                "content-type": "application/boschni-json-merger+json",
```

Comment:
are we using this custom content type to be able to merge directly objects (the cell sets) with the json (the stored celll sets) ?

Reply:
yes, it's a way to update just part of the cell sets, ie the clusters, irrespective of the content the user produced such as custom annotations